openthread/lib/cortex-m4/soft-float/v1.1/ftd/cc310/openthread_lib_configuration.txt

```diff
@@ -40,3 +46,4 @@ CONFIG_OPENTHREAD_THREAD_VERSION=1.1
 CONFIG_OPENTHREAD_THREAD_VERSION_1_1=y
 CONFIG_OPENTHREAD_UDP_FORWARD=y
 CONFIG_OPENTHREAD_XPANID=de:ad:00:be:ef:00:ca:fe
+OT_NCP_VENDOR_HOOK_SOURCE_DIR=/home/pisz/repo/ncs/nrf/samples/openthread/ncp
```

Comment:
This local folder path shouldn't go upstream.

Reply:
Done